.travis.yml

```diff
@@ -4,7 +4,7 @@
 
 language: cpp
 os: linux # Jobs are on Linux unless specified otherwise
-dist: trusty # Jobs are on Trusty unless specified otherwise
+dist: xenial # Jobs are on Xenial unless specified otherwise
```

Comment:
Why only `xenial`? You could use for example `focal` to get even more modern compiler and so on.